### ISSUE TITLE
The Youtube video doesn't seem to be opening！

### ISSUE THREAD

**Issue by MassSmith** [nfriedly/node-unblocker#269]
It seems that the Youtube video cannot be opened. 
Is there a problem with the component ytdl-core?

**Comment by nfriedly** [comment]
Yeah, I just tested it for https://github.com/nfriedly/nodeunblocker.com/issues/47 - and, yeah, my best guess is that google is now detecting and blocking ytdl-core.

**Comment by MassSmith** [comment]
Is there a good solution to this problem?

**Comment by nfriedly** [comment]
Maybe, but I don't know what it is
